tests/samples/expected/tree-sitter-query.scm

```diff
@@ -0,0 +1,1508 @@
+; Configuration
+(#language! ocaml)
```

Comment:
In the future, having such a large example file may prove unwieldy. We'll see, but if it gets difficult to review style changes, we should probably consider breaking it up in smaller chunks.